frontend/app/routes/login.tsx

```diff
@@ -45,6 +55,9 @@ export default function Login() {
         }}
       >
         <Form method="post">
+          <Typography variant="h2" sx={{ margin: "515px 0 25px 26px", textAlign: "center" }}>
+            Please Login:
+          </Typography>
```

Comment:
![image](https://user-images.githubusercontent.com/36668331/209407953-79eeecdf-7e6d-4349-8293-aa1576c01fb9.png)

Too many magic numbers, why do we need these?

Reply:
I will say yes because the design is fitted to image unfortunately

Reply:
Can't we use flexboxes, just like on the landing page?